cli/internal/cache/cache.go

```diff
@@ -62,6 +63,16 @@ type Opts struct {
 	Workers        int
 }
 
+var _remoteOnlyHelp = `Ignore the local filesystem cache for all tasks. Only
+allow reading and caching artifacts using the remote cache.`
+
+// AddFlags adds cache-related flags to the given FlagSet
+func AddFlags(opts *Opts, flags *pflag.FlagSet, repoRoot fs.AbsolutePath) {
+	// skipping remote caching not currently a flag
+	flags.BoolVar(&opts.SkipFilesystem, "remote-only", false, _remoteOnlyHelp)
+	fs.AbsolutePathVar(flags, &opts.Dir, "cache-dir", repoRoot, "Specify local filesystem cache directory.", "./node_modules/.cache/turbo")
```

Comment:
I might be missing something, why do windows users need to use unix separators?

Reply:
If we're running things in CI we should generally encourage consistent input otherwise the user can receive surprising output. I think that if we actually push path types through the entire system then we'd be okay, but right now I worry that there are edge cases is the user specifies Windows paths.

Reply:
Hmm. We have a few different kinds of paths. Cache directory is pretty clearly machine-local, and thus seems like it should be strictly platform-dependent. Especially since I don't think it can or should be configured via shared config file. 

Other paths are shared, for instance, global dependency paths. In my mind those are two different types (the global deps one is really a glob, even if you use a file path), but I'm sure we don't do a great job calling that out or helping with that distinction.

Actually, perhaps those are the only two in the user input? Absolute-or-relative platform-dependent path for things like cache directory, and unix-like glob for any paths shared across machines? Might be some good candidates for at least some internal types, if not external documentation.

In the CI case, I don't think it would make sense to share the same input value for cache directory across a linux machine and windows machine. Unless there's a windows convention I'm not aware of, I wouldn't expect a unix path to work to point to a local directory.